universal-application-tool-0.0.1/app/auth/saml/InvalidSamlProfileException.java

```diff
@@ -0,0 +1,8 @@
+package auth.saml;
+
+public class InvalidSamlProfileException extends RuntimeException {
```

Comment:
Per the style guide all public things should have a javadoc, even if it's just a simple one.